src/clojure_tic_tac_toe/computer_move.clj

```diff
@@ -1,10 +1,13 @@
 (ns clojure-tic-tac-toe.computer_move
   (:require [clojure-tic-tac-toe.game_handler :as game_handler]
-            [clojure-tic-tac-toe.utilities :refer [set-to-list-or-nil]]))
+            [clojure-tic-tac-toe.minimax :as minimax]))
 
 (defn make-random-move
-  [game-state]
-  (let [available-moves (game_handler/get-available-moves game-state)
-        unsorted-list (set-to-list-or-nil available-moves)]
-    (rand-nth unsorted-list)))
+  [{:keys [board]}]
+  (let [sorted-moves (game_handler/get-sorted-moves board)]
+    (rand-nth sorted-moves)))
+
+(defn make-minimax-move
+  [{:keys [board player]}]
+  (:move (minimax/minimax-move-and-score board player)))
```

Comment:
The score is concept internal to the minimax algorithm.  Your public API should not be returning it.

Reply:
The public function now only returns the move (rather than getting it from the move-and-scores map).

**Note**: the only thing, is that I've added the public `minimax-move` function to `minimax.clj`, but I've still kept all the tests for `minimax-move-and-score`, which is not being used (so I guess it technically shouldn't be public). However, I just wanted to keep the tests for now.